.github/workflows/cli.yml

```diff
@@ -0,0 +1,42 @@
+name: create-catalyst
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+  merge_group:
+    types: [checks_requested]
+
+jobs:
+  lint-typecheck:
+    name: Run Tests
+
+    runs-on: ubuntu-latest
+
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+      BIGCOMMERCE_STORE_HASH: ${{ secrets.BIGCOMMERCE_STORE_HASH }}
```

Comment:
Just added it, but make sure to add this https://github.com/bigcommerce/catalyst/blob/main/.github/workflows/basic.yml#L20

Reply:
Thank you!